### DISCUSSION TITLE
Reset password

### DISCUSSION THREAD

**Discussion by hamperfait** [simonwep/ocular#51] [Q&A] [answered]
I installed Ocular and have been using it for a while via docker compose. Problem is I lost the initial created password and can't log back in now. Is there any way to view the password and/or reset it so that I can keep using Ocular without having to re-enter all my data? Thank you!

**Comment by simonwep** [comment]
Hey! Unfortunately this is currently not possible, this really is an MVP. This would've to be done on the server-side, setting a new password because it's stored in a hashed form :/ I'm sorry that this happened for you. I've created an issue over at genesis (https://github.com/simonwep/genesis/issues/5) and will have a look into it though! 

I also added a [note](https://github.com/simonwep/ocular-docker?tab=readme-ov-file#first-time-setup) about this in the docker repository.

**Comment by hamperfait** [comment]
Thanks, I appreciate that nonetheless. Good thing I had just started, so I didn't lose too much data. Will take a while to add everything again, but still doable. Thanks!

**Comment by simonwep** [comment, marked as answer]
This is now possible with the latest version! Check out [admin controls](https://github.com/simonwep/ocular-docker?tab=readme-ov-file#admin-controls) :) Make sure to check out [migrating to a new version](https://github.com/simonwep/ocular-docker?tab=readme-ov-file#migrating-to-a-new-version).
